src/actions/mermActions.js

```diff
@@ -34,49 +33,6 @@ export function createMerm(mermDetails, callback) {
   };
 }
 
-export function getMerms() {
-  return dispatch => {
-    dispatch({ type: ActionTypes.REQUEST_INITIATED });
-
-    API.getMerms().then(
-      response => {
-        dispatch({
-          type: ActionTypes.UPDATE_MERM_LIST,
-          value: response.data
-        });
-        dispatch({ type: ActionTypes.REQUEST_SUCCESS });
-        dispatch({ type: ActionTypes.RESET_REQUEST_STATUS });
-      },
-      error => {
-        if (error["response"].errors[0].message == "User not signed in") {
-          dispatch({ type: ActionTypes.CLEAR_USER_OBJECT });
-          history.push("/login");
-          dispatch({
-            type: ActionTypes.SHOW_NOTIFICATION,
-            value: {
-              show: true,
-              type: "Error",
-              message: "Not Signed It!",
-              description:
-                "Looks like your token has expired, please sign in again!"
-            }
-          });
-        } else {
-          dispatch({
-            type: ActionTypes.SHOW_NOTIFICATION,
-            value: {
-              show: true,
-              type: "Error",
-              message: "Unable fetch Merms",
-              description: "Something is broken!"
-            }
-          });
-        }
-      }
-    );
-  };
-}
-
```

Comment:
We now get dashboard merms from elasticsearch and no longer require this action!